codecov.yml

```diff
@@ -0,0 +1,16 @@
+coverage:
+  precision: 2
+  round: down
+  range: "70...100"
+  status:
+    project:
+      default:
+        target: auto
+        threshold: 0.01
+    patch: false
+    changes: false
+  # ignore:
+    # - "examples/download_preprocess_finance.py"
```

Comment:
TODO: Remove.
One thing that should be ignored is the tests directory. It is also possible to ignore it from pycov (i.e. in pytest.ini:
```
[coverage:run]
omit =
    # Omit tests from measurement
    */tests/*
```

which is probably better anyways

Reply:
Tests files should have a coverage of 100 % by design, shouldn't they ?

Reply:
Yes (minus the GPU-only tests which don't run). It's just useless to show them with codecov

Reply:
done